extensions/amp-story/1.0/test/test-amp-story-embedded-component.js

```diff
@@ -229,6 +237,79 @@ describes.realWin('amp-story-embedded-component', {amp: true}, env => {
       expect(tooltipTextEl.textContent).to.equal('google.com');
     });
   });
+
+  it('should send data-var specified by publisher in analytics event', () => {
+    const analyticsSpy = sandbox.spy(analyticsService, 'getDetails');
```

Comment:
This is a weird one to spy on, can you test what's passed to `triggerAnalyticsEvent` or to your handler instead? This way you test things more end to end.

Reply:
`triggerAnalyticsEvent` is a standalone utility function and Sinon doesn't support spying it unfortunately :( https://github.com/sinonjs/sinon/issues/562 . 

Regarding the handler: which one do you mean? The one added in `AmpStoryEventTracker`?